includes/admin/class-sensei-setup-wizard.php

```diff
@@ -234,16 +238,39 @@ public function render_wizard_page() {
 	 *
 	 * @return boolean
 	 */
-	private function should_current_page_display_wizard() {
-		$screen = get_current_screen();
+	private function should_current_page_display_wizard_notice() {
+		// Dashboard, plugins, and Sensei pages, except Sensei Home.
+		$screens_without_sensei_prefix = [
+			'dashboard',
+			'plugins',
+			'edit-sensei_message',
+			'edit-course',
+			'edit-course-category',
+			'admin_page_course-order',
+			'edit-module',
+			'admin_page_module-order',
+			'edit-lesson',
+			'edit-lesson-tag',
+			'admin_page_lesson-order',
+			'edit-question',
+			'question',
+			'edit-question-category',
+		];
 
-		if ( false !== strpos( $screen->id, 'sensei-lms_page_sensei' ) ) {
-			return true;
-		}
+		return $this->check_sensei_screen( $screens_without_sensei_prefix );
+	}
 
+	/**
+	 * Check if current screen is selected to redirect to the wizard.
+	 *
+	 * @return boolean
+	 */
+	private function should_current_page_redirect_to_wizard() {
```

Comment:
I separated the checks for the notice and the redirect because the notice is very ugly in Sensei Home and we want the redirect if the user navigates to the Sensei Home.